docs/node/quickstart.mdx

```diff
@@ -21,6 +25,12 @@ genesis file, private key, and more in the `~/.kwild` directory.
 kwild --autogen
```

Comment:
Does kwild need flags for postgres?

Reply:
The defaults will work with our docker compose services, or just with the docker image after https://github.com/kwilteam/kwil-db/pull/557

@Yaiba Was thinking quickstart could have a `run.sh` script we pull from a url.  I feel like running `kwild` from the command line is more the interaction we want here, but that's up for debate.